clients/vscode/src/extension.ts

```diff
@@ -85,8 +84,6 @@ export async function activate(context: ExtensionContext) {
       webviewOptions: { retainContextWhenHidden: true },
     }),
   );
-  // Create chat panel view
-  new ChatPanelViewProvider(context, client.agent, gitProvider, client.chat);
```

Comment:
It seems like this line is removed in the main branch. Maybe there is a conflict causing it to be added back?